pyramid_urireferencer/protected_resources.py

```diff
@@ -33,16 +32,6 @@ def protected_operation(fn):
     def advice(parent_object, *args, **kw):
         referencer = pyramid_urireferencer.get_referencer(parent_object.request.registry)
         uri = referencer.get_uri(parent_object.request)
-        if uri is None:
```

Comment:
Not sure where this code went. But we should not raise a 501 when the urireferencing fails. For the client this is an HTTP 500 Internal Server Error and nothing more. We should have some good logging for the server admin though.